internal/cli/atlas_alert_config.go

```diff
@@ -20,11 +20,13 @@ import (
 
 func AtlasAlertConfigBuilder() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "alert-config",
-		Aliases: []string{"alertConfig", "alertConfigs", "alert-configs"},
+		Use:     "alerts configs",
```

Comment:
if you wanted to do this now, please create a file `internal/cli/atlas_alerts.go` and define `alerts` there and add the config command to that one
Also only add `alert` as an alias

Reply:
once you create that file this should only be `configs` and only add `config` as an alias